subsys/bluetooth/mesh/rpl_storage.h

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) 2017 Intel Corporation
+ * Copyright (c) 2020 Lingao Meng
+ * Copyright (c) 2021 Nordic Semiconductor ASA
```

Comment:
If this is a new file why did you add Intel and Lingao? :) and old year?

Reply:
To be honest im quite blank on which changes justifies changing the copyright stuff 😅